neo/Network/Message.cs

```diff
@@ -129,9 +138,9 @@ private static uint GetChecksum(byte[] value)
         void ISerializable.Serialize(BinaryWriter writer)
         {
             writer.Write(Magic);
-            writer.WriteFixedString(Command, 12);
+            writer.WriteFixedString(Command.ToString(), 12);
```

Comment:
what is the output of `Command.ToString()`?  

Would, for example a MessageCommand.GetData be output as `getdata`, or `9` ?

Reply:
It should be the string representation of the `enum` value name string
For example: `Command. version` => `"version"`

Reply:
This ensures compatibility and reduces the risk of errors.

Reply:
But my main intention is to propose a protocol change later on. So that instead of sending a 12-byte string, only 1 byte representing its value of the listed type is sent.

Reply:
> Would, for example a MessageCommand.GetData be output as getdata, or 9 ?

Reply:
Hi, can you revert the changes to Message? I think it should be in a separate PR, and be applied in future.